frontend/src/app/home-page/home-page.component.ts

```diff
@@ -1,15 +1,34 @@
 import { Component, OnInit } from '@angular/core';
 
+interface Item {
+  h1Title: string,
+  h2Title: string,
+  text: string,
+  buttonText: string,
+}
+
+interface ItemImage {
+  src: string,
+}
+
 @Component({
   selector: 'app-home-page',
   templateUrl: './home-page.component.html',
   styleUrls: ['./home-page.component.css']
 })
 export class HomePageComponent implements OnInit {
+  /* Breakpoint is the number of columns that is shown. */
+  breakpoint: number = 2;
+  maxWidth: number = 600;
```

Comment:
make it public, if you're using it in the html, or private if not

Reply:
This will disappear in the next pr